lib/job.js

```diff
@@ -86,6 +87,18 @@ class Job extends BaseModel {
     }
 
     /**
+     * Return PR number. Returns -1 if this is not a PR job
+     * @return {Number}     PR number
+     */
+    prNum() {
+        if (!this.isPR()) {
+            return -1;
```

Comment:
Can we make this into a getter that is stored on first access?  Otherwise the sort function below will be more expensive then it needs to be.